---
permalink: /
title: "William Pearman"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Website under development
======

About me
======
I'm a PhD student with Associate Professor [Ceridwen Fraser](http://ceridwenfraser.com/), Dist. Prof. [Neil Gemmell](https://gemmell-lab.otago.ac.nz/), and Dr [Sergio Morales](https://micro.otago.ac.nz/our-people/teaching-research-and-support/sergio-morales/) at the University of Otago, Dunedin. My PhD research focuses on large macroalgae around New Zealand, examining the adaptive and bigeographical significance of the microbiome - trying to understand how genetic, environmental, and geographic factors can interact to shape the microbiome.

Prior to starting at the University of Otago, I was based at Massey University in Albany, New Zealand. I completed my Bachelors and Master degrees there, researching population genomics and mitochondrial genome structure in Isocladus armatus, an endemic isopod. I worked with Prof. James Dale, Dr Nikki Freed, and Dr Sarah Wells.

Research Summary
======
Prior to joining the University of Otago, I worked on a range of research projects at Massey University.
I worked with Libby Liggins, Dave Aguirre, and Vanessa Arranz to develop a pipeline to create [custom metabarcoding databases](https://github.com/wpearman1996/MARES_database_pipeline) - called [MARES](https://www.nature.com/articles/s41597-020-0549-9). For this project we developed a set of tools and built on some of those previously developed to create databases of COI sequences for marine metabarcoding. 

During this time, I was also involved in the [Ira Moana project](https://sites.massey.ac.nz/iramoana/), developing analysis pipelines for re-use of genetic data and helping to compile metadata from previous publications. 

While I completed my undergraduate and masters degree, I also studied the role that read length plays in characterizing communities with [metagenomics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3528-4). For this research, we found that community characterization success depended critically on i) kingdom ii) read length, and iii) sequencing technology.
